packages/core/src/components/Footer/Footer.tsx

```diff
@@ -30,43 +26,37 @@ export const HvFooter = (props: HvFooterProps) => {
     name = "Hitachi Vantara",
     copyright = `© Hitachi Vantara Corporation ${new Date().getFullYear()}. All Rights Reserved.`,
     links,
-    classes,
+    classes: classesProp,
     className,
     ...others
   } = useDefaultProps("HvFooter", props);
   const muiTheme = useTheme();
+  const { classes, cx } = useClasses(classesProp);
+
+  const isSmDown = useMediaQuery(muiTheme.breakpoints.down("sm"));
 
   return (
-    <StyledRoot
-      $breakpoints={muiTheme.breakpoints}
-      className={clsx(className, footerClasses.root, classes?.root)}
+    <footer
+      className={cx(className, classes.root, isSmDown && classes.small)}
```

Comment:
```suggestion
      className={cx(classes.root, className, {[classes.small]: isSmDown})}
```
- `className` needs to come after our internal styles (ie. `classes.root`)
- the `{[value]: condition}` `sx` syntax is preferable as it's easier to maintain when adding more conditionals (also, consistency).

This `classes.small` is a new class that will be exposed to the users now. (I'm not against it), but overriding `classes.small` would mean overriding multiple (unrelated) elements. We should keep `classes.X` on a single element type, otherwise it's confusing and not useful when overriding. We should only need `classes.small` here in the root